netsuite/10.0/modules/ROOT/pages/netsuite-studio-configure.adoc

```diff
@@ -76,193 +76,49 @@ To configure the global element for Netsuite Connector, follow these steps:
 
 . Select the name of the connector in the Studio canvas.
 . In the configuration screen for the operation, click the plus sign (+) next to the *Connector configuration* field to access the global element configuration fields.
-. On the *General* tab, select an authentication mechanism and configure it as described:
-+
-[%header,cols="30s,70a"]
-|===
-|Configuration |Description
-|Request Based Authentication |Using this strategy, you can send user credentials in the SOAP header of each request instead of authenticating to NetSuite by invoking login.
-|Token Authentication | This is a token-based authentication system that uses a consumer and token key or secret pairs, which are established within the NetSuite environment.
-|===
-+
-. Configure these fields according your selection:
-.. Request Based Authentication
-+
+. On the *General* tab, select an authentication mechanism and configure it as described: +
+* <<request-based-authentication,Request Based Authentication>>
+* <<token-authentication,Token Authentication>>
+. On the *Advanced* tab, optionally specify reconnection information, including a reconnection strategy.
+. Click *Test Connection* to confirm that Mule can connect with the specified server.
+. Click *OK*.
+
+[[request-based-authentication]]
+=== Request Based Authentication
+
+Use this strategy to send user credentials in the SOAP header of each request instead of authenticating to NetSuite by invoking login.
+
 image::netsuite-request-studio.png[request based authentication]
 +
 [%header,cols="30s,70a"]
 |===
 |Parameter |Description
-|Name |Enter a name for the configuration with which to reference it later.
-|Email |Enter the login email of the NetSuite UI.
-|Password |Enter the corresponding password to log into NetSuite UI.
-|Account |Enter the account ID of the SuiteTalk NetSuite web services. This is usually found within NetSuite sandbox UI under Setup > Integration > Web Service Preferences.
+|Name |Enter a name for the configuration to reference in subsequent configurations of NetSuite Connector instances.
+|Email |Enter the email used to log in to the NetSuite UI.
+|Password |Enter the corresponding password to log in to NetSuite UI.
+|Account |Enter the account ID of the SuiteTalk NetSuite Web Services. This is usually found within the NetSuite sandbox UI under *Setup > Integration > Web Service Preferences*.
```

Comment:
```suggestion
|Account |Enter the account ID for SuiteTalk NetSuite Web Services. You can find this within the NetSuite sandbox UI under *Setup > Integration > Web Service Preferences*.
```

Reply:
Should knowing the ID be a prereq., so users don't get to this step and have to backtrack?

Reply:
It was kinda hard to do this considering the way the topic is structured. See what you think of how I handled it.